### PR TITLE
[air] Add xgboost release test for silver tier(10-node case).

### PR DESCRIPTION
**Pull request by xwjiang2010**
## Why are these changes needed?
Add release test for air xgboost benchmark.
Will take a look at the number first running in release test environment and add a proper triggering threshold (time * 1.1).

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by xwjiang2010** [comment]
Another run: https://buildkite.com/ray-project/release-tests-pr/builds/8990#0181f2ef-b0be-4884-aca2-98f0cd3ec00c

**Review by Yard1 — APPROVED** [review]
LGTM thanks!

**Comment by xwjiang2010** [comment]
@Yard1 Do you know why the result shows
```
training_time = 1850.115315906
prediction_time = 86.38269096899967
```

**Comment by xwjiang2010** [comment]
> nice! How do we get this onto the dashboard?

waiting for Kai to set something up in his PR and will just follow the practice.

**Comment by Yard1** [comment]
Actually to be 100% sure can we run the training part and prediction part as separate subprocesses?

**Comment by Yard1** [comment]
also @xwjiang2010 we are missing the EBS config, I think that may be a cause for slowdown

**Comment by Yard1** [comment]
@xwjiang2010 we probably can bring down the disk size to like 200 GBs to save a little money!

**Comment by xwjiang2010** [comment]
@Yard1 haha sounds good :) 
do you know how can I know how much is being spilled?

**Comment by Yard1** [comment]
I think you can use this? https://docs.ray.io/en/latest/ray-core/objects/object-spilling.html#stats

**Comment by xwjiang2010** [comment]
```
training_time = 1663.45228263
prediction_time = 77.18809864599962
```
@Yard1

File: release/air_tests/air_benchmarks/xgboost_app_config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - curl
+
+python:
+  pip_packages:
+    - pytest
+  conda_packages: []
+
+post_build_cmds:
+  - pip3 uninstall ray -y || true && pip3 install -U {{ env["RAY_WHEELS"] | default("ray") }}
+  - pip3 install -U --force-reinstall --no-deps xgboost xgboost_ray  # Avoid caching
```
Comment (Yard1) [inline release/air_tests/air_benchmarks/xgboost_app_config.yaml:13]:
```suggestion
  - pip3 install -U --force-reinstall --no-deps xgboost_ray  # Avoid caching
```

**Comment by Yard1** [comment]
That should be fine, thanks

**Pull request merged**